admin/src/main/resources/io/buoyant/admin/js/src/delegator.js

```diff
@@ -54,9 +54,11 @@ define([
         e.preventDefault();
         var path = $('#path-input').val();
         window.location.hash = encodeURIComponent(path);
-        var request = $.get(
-          "delegator.json?" + $.param({ path: path, dtab: dtabViewer.dtabStr(), namespace: namespace }),
-          renderAll.bind(this));
+        var request = $.post(
```

Comment:
I checked this out and tested it in the UI - since we're now checking for content type json on the server, as is, I get a `POST http://localhost:9990/delegator.json 415 (Unsupported Media Type)` error when I try to test a delegation... so we'll need to switch this to use `$.ajax`  so we can specify a content type:

```
        var request = $.ajax({
            url: 'delegator.json',
            method: 'POST',
            contentType: 'application/json',
            data: JSON.stringify({ namespace: namespace, dtab: dtabViewer.dtabStr(), path: path })
        }).done(renderAll.bind(this));
```